### ISSUE TITLE
预训练权重

### ISSUE THREAD

**Issue by rabbit218218** [LMD0311/PointMamba#45]
I have noticed that your pretrained weights are labeled with different names. Were these obtained through training on different datasets or using different methods? When I performed the pretraining myself and then used the pretrained weight files for downstream tasks (ModelNet40 classification), the accuracy in the first epoch was around 8 and then gradually increased. However, when I used your weight files for classification, the accuracy in the first epoch was over 93. I don't understand why this is happening. Could you please explain?

**Comment by rabbit218218** [comment]
![Image](https://github.com/user-attachments/assets/cabfc87f-1a36-4e29-941f-bc648d1f568b)

**Comment by LMD0311** [comment]
```pretrain.pth``` is the pretrained weight. ```modelnet_pretrain.pth``` uses the pretrained weight and finetunes on ModelNet. ```modelnet_scratch.pth``` finetunes on ModelNet from random initialization without using pretrained weights.
